CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (<https://github.com/openvinotoolkit/datumaro/pull/912>)
 - Add tfds:uc_merced and tfds:eurosat download
   (<https://github.com/openvinotoolkit/datumaro/pull/914>)
+- Refactor Media
```

Comment:
could you add more descriptions such as `Refactor Media to be initialized from one of path, data and byte`

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/921/commits/6e430dc126b6351de3204cb59468c87b5ae2cc9e